phoenicis-scripts/src/main/java/org/phoenicis/scripts/nashorn/NashornEngine.java

```diff
@@ -24,23 +24,23 @@
 import java.util.List;
 import java.util.function.Consumer;
 
-class NashornEngine {
+public class NashornEngine {
     private final ScriptEngine scriptEngine;
     private final List<Consumer<Exception>> errorHandlers = new ArrayList<>();
 
     NashornEngine(ScriptEngine scriptEngine) {
         this.scriptEngine = scriptEngine;
     }
 
-    void eval(InputStreamReader inputStreamReader, Consumer<Exception> errorCallback) {
+    public void eval(InputStreamReader inputStreamReader, Consumer<Exception> errorCallback) {
         try {
             this.scriptEngine.eval(inputStreamReader);
         } catch (Exception e) {
```

Comment:
Hum, in this particular case, it is right to catch all exception, because we cannot predict what exception will be thrown in the script.